### ISSUE TITLE
half of the page gets cut

### ISSUE THREAD

**Issue by aaayushsingh** [navjotdhanawat/dynamic-html-pdf#12]
The handlebars page renders fine in browser, but when creating pdf using this lib, the page is cut in half horizontally. So, around 40% of the right side of the page is not visible in pdf

**Comment by navjotdhanawat** [comment]
@aaayushsingh Is this issue resolved?

**Comment by aaayushsingh** [comment]
@navjotdhanawat Not really. I tried different page sizes until I found the one that fit
